### PR TITLE
FHIR-37116

### PR DESCRIPTION
**Pull request by bvdh**
https://jira.hl7.org/browse/FHIR-37116

### CODE REVIEW

File: input/pagecontent/4-2-syncconsiderations.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ Many applications go into edit mode or start a modal dialog that locks the syste
 |System|Failure mode|Possible actions|
 |--|--|--|
 | Subscriber | Modal dialog open in UI, unable to change case without losing end user data | Present end user with clear indication that contextual synchronization is lost. Respond with a http status code of 409 conflict. |
-| Subscriber | Unable to change context | Respond with a http status code of 409 conflict. |
+| Subscriber | Unable to change context | Responds with a 202 Accepted and sends a syncerror when the context change is refused, stating the source and reason for change. |
```
Comment (EricOnFHIR) [inline input/pagecontent/4-2-syncconsiderations.md:91]: This does not seem to be consistent with the Resolution Description in the JIRA ticket:

Subscribing Client | Unable to change context | Client responds with a http status code of 409 conflict. If client is unable to determine inability to follow context, it responds with a 202 Accepted and sends a syncerror when the context change is refused, stating the source and reason for change

Comment (isaacvetter) [inline input/pagecontent/4-2-syncconsiderations.md:91]:
```suggestion
| Subscriber | Unable to change context | Client responds with a http status code of 409 conflict. If client is unable to determine inability to follow context, it responds with a 202 Accepted and sends a syncerror when the context change is refused, stating the source and reason for change. |
```

**Review by isaacvetter — APPROVED** [review]

**Comment by isaacvetter** [comment]
@EricOnFHIR , do you want to re-review this?

**Comment by isaacvetter** [comment]
Merging because jira is already voted upon. Marking jira as applied.

**Pull request merged**